Tools/TestWebKitAPI/Tests/WebKitCocoa/MediaLoading.mm

```diff
@@ -134,9 +135,7 @@ HTTPServer manifestServer([&](Connection connection) mutable {
 static Vector<uint8_t> testVideoBytes()
 {
     NSData *data = [NSData dataWithContentsOfURL:[[NSBundle mainBundle] URLForResource:@"test" withExtension:@"mp4" subdirectory:@"TestWebKitAPI.resources"]];
-    Vector<uint8_t> vector;
-    vector.append(static_cast<const uint8_t*>(data.bytes), data.length);
-    return vector;
+    return toSpan(data);
```

Comment:
Should be `toVector` instead of `toSpan`. Also I think this could just be a one-liner.